admin/google_search_console/class-crawl-category-issues.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ */
+
+/**
+ * Class WPSEO_Crawl_Category_Issues
+ */
+class WPSEO_Crawl_Category_Issues {
+
+	/**
+	 * @var string
+	 */
+	private $category;
+
+	/**
+	 * @var WPSEO_GWT_Service
+	 */
+	private $service;
+
+	/**
+	 * @var string
+	 */
+	private $platform;
+
+	/**
+	 * @var string
+	 */
+	private $option_name = '';
+
+	/**
+	 * List of all current issues to compare with received issues
+	 *
+	 * @var array
+	 */
+	private $current_issues = array();
+
+	/**
+	 * Holder for all the issues
+	 *
+	 * @var array
```

Comment:
Because we know this is an array of `WPSEO_Crawl_Issue`s we can use `WPSEO_Crawl_Issue[]` as the `@var`. The same is true for all other places we are passing `WPSEO_Crawl_Issue`s. This makes the code a lot clearer and we get better IDE autocomplete. 😃